src/evaluators/response/QCAD_ResponseFieldValue.hpp

```diff
@@ -126,6 +126,47 @@ namespace QCAD {
     int numNodes;
   };
 
+  template<typename Traits>
+  class FieldValueScatterScalarResponse<PHAL::AlbanyTraits::HessianVec,Traits> :
```

Comment:
Are we even using this evaluator anymore? I know all the work has been done already, so we might as well, but we might as well trim unused (and unlikely to be used) stuff...

Reply:
I have no strong feeling regarding this class and the other commented one.
I had to provide some implementation (even empty) to build and link correctly.
If they are not used, I assume that they can be removed. Should it be included in this PR or in another one?

Reply:
Perhaps another one. However, if these classes give you headaches later on, you may consider removing them. Unless @mperego thinks otherwise.

Reply:
Yeah, @bartgol let's look at those QCAD evaluators after this PR is merged in. We should either remove them or rename them depending on whether we used them or not. I can have a look at that.